templates/base.html

```diff
@@ -75,12 +75,12 @@
                             <li><a href="{% url 'faq' %}">FAQ</a></li>
                             <li><a href="{% url 'contact' %}">Contact</a></li>
                         </ul>
-                        <div id="socials">
-                            <a href="https://www.facebook.com/828188463946021" target="_blank"><img class="facebook" src="{% static "img/socials/facebook_logo.png" %}" width="30" height="30" /></a>
-                            <a href="http://instagram.com/perdiemmusic" target="_blank"><img class="instagram" src="{% static "img/socials/instagram_logo.png" %}" width="30" height="30" /></a>
-                            <a href="http://twitter.com/perdiemmusic" target="_blank"><img class="twitter" src="{% static "img/socials/twitter_logo.png" %}" width="30" height="30" /></a>
-                            <a href="https://www.youtube.com/channel/UC-6u9L-W6Dn7kaYLOz4YuTg" target="_blank"><img class="youtube" src="{% static "img/socials/youtube_logo.png" %}" width="30" height="30" /></a>
-                        </div>
+                            <ul class="social-icons">
```

Comment:
This whole list should be unindented 4 spaces so that it's aligned with the `"footer-items"` list class.